modules/images/dominant-color/load.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * Module Name: Dominant Color
+ * Description: Adds support to store dominant-color for an image and create a placeholder background with that color.
+ * Experimental: Yes
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+/**
+ * Add the dominant color metadata to the attachment.
+ *
+ * @since n.e.x.t
+ *
+ * @param array $metadata        The attachment metadata.
+ * @param int   $attachment_id   The attachment ID.
```

Comment:
Unnecessary indentation:
```suggestion
 * @param array $metadata      The attachment metadata.
 * @param int   $attachment_id The attachment ID.
```